moveit_setup_assistant/CMakeLists.txt

```
@@ -57,6 +57,7 @@ catkin_package(
     moveit_ros_planning
     moveit_ros_visualization
     roscpp
+    rviz
```

This one is not necessary. See discussion in https://github.com/ros-planning/moveit/pull/2116#issuecomment-635448407.
```suggestion
    rviz
```
However, given that MSA doesn't have downstream dependencies, it's better to add it here than to suppress the warning and risk ignoring future other stuff as well?